website/docs/r/pipes_pipe.html.markdown

```diff
@@ -86,7 +86,7 @@ resource "aws_sqs_queue" "target" {}
 resource "aws_pipes_pipe" "example" {
   depends_on = [aws_iam_role_policy.source, aws_iam_role_policy.target]
   name       = "example-pipe"
-  role_arn   = aws_iam_role.example.arn
+  role_arn   = aws_iam_role.test.arn
```

Comment:
We may want to flip this and change the resource name from `test` to `example` (I think this is the more common convention in the docs, though copy/paste from acceptance tests often leaks in).

Reply:
Totally agree; done 👍